site/docs/plugins/hydrate.md

```diff
@@ -1,4 +1,4 @@
-# Hydration Plugin for grammY
+# Hydration Plugin for grammY (`hydrate`)
```

Comment:
Nice catch, thank you!

Reply:
Same to files plugin, I add it in #86 too.